project-base/storefront/components/Pages/CategoryDetail/CategoryBestsellers/CategoryBestsellers.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button } from 'components/Forms/Button/Button';
+import { ListedProductFragmentApi } from 'graphql/generated';
+import { useGtmSliderProductListViewEvent } from 'gtm/hooks/productList/useGtmSliderProductListViewEvent';
+import { GtmProductListNameType } from 'gtm/types/enums';
+import { useComponentUpdate } from 'hooks/helpers/useComponentUpdate';
+import { useTypedTranslationFunction } from 'hooks/typescript/useTypedTranslationFunction';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import { CategoryBestsellersListItem } from './CategoryBestsellersListItem';
+
+const NUMBER_OF_VISIBLE_ITEMS = 3;
+
+type CategoryBestsellersProps = {
+    products: ListedProductFragmentApi[];
+};
+
+export const CategoryBestsellers: FC<CategoryBestsellersProps> = ({ products }) => {
+    const t = useTypedTranslationFunction();
+    const router = useRouter();
+    const [isCollapsed, setIsCollapsed] = useState(false);
+
+    useComponentUpdate(() => setIsCollapsed(false), [router.asPath]);
+    useGtmSliderProductListViewEvent(products, GtmProductListNameType.bestsellers);
+
+    if (products.length === 0) {
+        return null;
+    }
+
+    return (
+        <div className="mb-8">
+            <div className="mb-3 break-words font-bold text-dark lg:text-lg">
+                {t('Do not want to choose? Choose certainty')}
+            </div>
+            <div className="mb-5">
+                {products.map((product, index) => {
+                    return index <= NUMBER_OF_VISIBLE_ITEMS - 1 || isCollapsed ? (
+                        <CategoryBestsellersListItem
+                            key={product.uuid}
+                            product={product}
+                            gtmProductListName={GtmProductListNameType.bestsellers}
+                            listIndex={index}
+                        />
+                    ) : null;
+                })}
+            </div>
+            {products.length > NUMBER_OF_VISIBLE_ITEMS && (
+                <div className="text-center">
+                    <Button type="button" size="small" onClick={() => setIsCollapsed((prev) => !prev)}>
+                        <span>
+                            {isCollapsed
+                                ? t('Hide the next {{ count }} best sellers', {
+                                      count: products.length - NUMBER_OF_VISIBLE_ITEMS,
+                                  })
+                                : t('View the next {{ count }} best sellers', {
+                                      count: products.length - NUMBER_OF_VISIBLE_ITEMS,
+                                  })}
```

Comment:
I am not sure if it's just a bug on my side but I don't see the `count` variable shown in the translation string.

![image](https://github.com/shopsys/shopsys/assets/11465991/e2c3a06e-ea3d-438b-b5d6-084652fa3e39)

Reply:
This is an issue with translates which pisses me off. Not a problem with code, but since I removed the dynamic translations, I don't think we need to take care of it rn.